src/python/pants/backend/awslambda/python/rules.py

```diff
@@ -86,10 +87,15 @@ async def package_python_awslambda(
         ],
     )
 
+    lockfile_hex_digest = None
+    if lambdex.lockfile != "<none>":
```

Comment:
This magic string ended up peppered a lot of places. Would be good to have `def lockfile` convert `"<none>"` to `str | None` instead.

Reply:
Imo fine to do in a followup. This code path is still changing frequently.